.github/workflows/tests.yaml

```diff
@@ -258,6 +258,15 @@ jobs:
                         --copy-artifacts-to objdir-clone \
                      "
 
+            - name: Run Python chip controller Tests
```

Comment:
This should be using https://github.com/project-chip/connectedhomeip/blob/master/docs/testing/python.md#defining-the-ci-test-arguments to run automatically and there should be no separate manual run for it.